dist/alasql.js

```diff
@@ -14454,7 +14513,9 @@ alasql.into.CSV = function(filename, opts, data, columns, cb) {
 	data.forEach(function(d){
 		s += columns.map(function(col){
 			var s = d[col.columnid];
-			s = (s+"").replace(new RegExp('\\'+opt.quote,"g"),'""');
+			if (opt.quote !== '') {
+				s = (s+"").replace(new RegExp('\\'+opt.quote,"g"),'""');
```

Comment:
It seems to me that we need to clarify what the expected behavior is when `opt.quote` is defined to be a character that appears in the data strings.

For example, if `opt.quote = '?'` and one of the column values is the string `'test?me'`, do we want the output in the CSV file to be `?test""me?`?  If `opt.quote = '"'` and one of the column values is the string `'help"me'`, should the CSV output be `"help""me"`?  This is the current behavior.

With respect to the case where `opt.quote = ''`, the CSV output should always have the format of `"string goes here"`, correct?  This is because we're keeping compatible with Microsoft Excel.